src/app/components/header/header.component.html

```diff
@@ -20,5 +20,8 @@ <h5 class="offcanvas-title" id="offcanvasNavbarLabel">Menu</h5>
       </div>
     </div>
   </div>
+  <div class="logout-container">
```

Comment:
Should this always be visible? Felt weird seeing it before I had even logged in

Reply:
Maybe we can stick a logout button at the bottom of the sidebar menu

Reply:
I actually think we already have the code in place for this. The only thing I may need to modify is adding the option to pass a callback function instead of just a route to direct to

Reply:
Yeah should be pretty easy to move there if you want. Just go to header.component.ts and add to the menuLinks array:

`{ text: 'Logout', routerLink: 'whatever your login route is/will be', imgPath: 'assets/box-arrow-right.svg' }`

I figured out how to add a callback function should be pretty easy to do, I can make a PR for that real quick if you want. I also should make some image improvements, so you can set the alt text and also width and height.

Reply:
I made a branch for sidebar menu improvements. Added option for callback function and also added the described image improvements. Adding the img stuff made me realize how gross the sidebar-list.component, is I really want to rebuilt it now, it's hard to work with

Reply:
I have a PR up for the sidebar improvements. If we'd like, it would be easy to have a logout button in the sidebar menu